frontend/sentry.js

```diff
@@ -14,7 +14,8 @@ async function createReleaseAndUpload() {
     await cli.releases.uploadSourceMaps(release, {
       include: ['build/static/js'],
       urlPrefix: '~/static/js',
-      validate: true
+      validate: true,
```

Comment:
I think we can remove this, refer Sentry-cli [docs](https://docs.sentry.io/product/cli/releases/#sentry-cli-sourcemaps) on uploading source maps

Regarding the default rewrite option
> It automatically validates source maps before upload very accurately which can spot errors you would not find otherwise until an event comes in. This is an improved version of what --validate does otherwise.

Regarding the validate option 
> This attempts source map validation before upload when **rewriting is not enabled.**

Reply:
Oh I see, so we can remove both `validate: true` and `rewrite: true`? That sounds good, but would we want to keep the options simply for clarity?

Reply:
Oh I see. Alright, I've removed it.